emission/net/ext_service/geocoder/nominatim.py

```diff
@@ -166,37 +166,51 @@ def return_address_from_location_google(lat, lon):
         #This try block is for our first 150,000 requests. If we exceed this, use Jack's Token.
         key_string = '&key=' + GOOGLE_MAPS_KEY
         url = base_url + latlng + key_string #Builds the url
+        print(url)
         result = requests.get(url).json() #Gets google maps json file
         cleaned = result['results'][0]['address_components']
-
-
         #Address to check against value of check_against_business_location
         chk = cleaned[0]['long_name'] + ' ' + cleaned[1]['long_name'] + ', ' + cleaned[3]['long_name']
         business_tuple = check_against_business_location(lat, lon, chk)
-        
+        location_is_service = isLocationService(cleaned)
+
         if business_tuple[0]: #If true, the lat, lon matches a business location and we return business name
             address_comp = cleaned[0]['long_name'] + ' ' + cleaned[1]['short_name']
-            return business_tuple[1], cleaned[3]['short_name'], address_comp
+            #, cleaned[3]['short_name'], address_comp
+            return business_tuple[1], location_is_service
```

Comment:
I don't think you should change what is returned, you can add on another result though

Reply:
I changed what should be returned because I looked through the code and saw that I never actually used cleaned[3]['short_name'] or address_comp for the rest of my code.

Reply:
@valin1 that might be true, but it is very short-term. What if we wanted to merge this into master at some point? What if another group wanted to re-use it? The nominatim file is really shared code, isn't it?